systemvm/debian/opt/cloud/bin/setup/bootstrap.sh

```diff
@@ -29,124 +29,6 @@ log_it() {
   log_action_msg "$@"
 }
 
-hypervisor() {
-  if [ -d /proc/xen ]; then
```

Comment:
great to see this removed, quick question @Pearl1594 - would that cause an issue - we don't want hypervisor specific logic to be kicked or was this moved?